addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -67,6 +67,33 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {mail.messaging}
+         */
+        _computeMessaging() {
+            return [['link', this.env.messaging]];
+        }
+
+        /**
+         * @private
+         * @returns {String[]}
+         */
+        _computeTextInputSendShortcuts() {
+            if (!this.thread) {
+                return;
+            }
+            // The goal here is to target small screen size and not actual mobile device
+            // to allow the sending of message with shortcut.
```

Comment:
This comment should only concern the `isMobile` line, not the part of the condition about `mass_mailing`. I would actually split the 2 conditions in different returns just to make it clear.

I would actually also explain the bug in this comment: why is it necessary to make an exception for small screen? How comes the normal "enter" to send doesn't work? Btw I'm curious to have the answer because I'm still not sure to understand.

Reply:
@seb-odoo 
because of this condition (https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/components/composer_text_input/composer_text_input.js#L282) 'enter' didn't work to send a message for a small screen.